cabal.project

```diff
@@ -64,3 +64,36 @@ source-repository-package
   type: git
   location: https://github.com/jaccokrijnen/plutus-cert
   tag: e814b9171398cbdfecdc6823067156a7e9fc76a3
+
+if impl(ghc >= 9.8)
+  allow-newer:
+    , aeson:th-abstraction
+    , dependent-sum:some
+    , hedgehog-fn:hedgehog
+    , hedgehog:primitive
+    , lens:th-abstraction
+    , template-haskell:pretty
+
+    , cardano-binary:base
+
+    -- https://github.com/aisamanra/config-ini/pull/47
+    , config-ini:*
+
+    -- https://github.com/obsidiansystems/constraints-extras/pull/53
+    , constraints-extras:*
+
+    -- https://github.com/haskell-unordered-containers/hashable/pull/289
+    , hashable:containers
+
+    -- https://github.com/typeclasses/hex-text/pull/8
+    , hex-text:*
+
+    -- https://github.com/Gabriella439/turtle/pull/449
+    , turtle:*
+
+  constraints:
+    , brick >= 1.10
```

Comment:
these should go in the cabal files I guess? I suspect in many cases it's fine to unconditionally put these lower bounds in.

Reply:
These are currently needed while I have a whole bunch of `allow-newer` stanzas. I expect them to disappear once they are ready to be removed,